docs/xdr/usecases/playbook/ExtractIP_from_Url_country.md

```diff
@@ -0,0 +1,40 @@
+# Extract IP from url & Country  
+
+This use case describes how to extract an IP address from a URL and a country.  
+
+
+## Prerequisites
+
+- A SEKOIA.IO XDR licence
+- The user that configures the Playbook should have a role that contains:
+	* The SYMPHONY permissions
+- An API Key with a role that contains at least the following permission:
+	* `SIC_READ_ALERTS`
+
+> To create your API Key, follow this [documentation](../../../getting_started/generate_api_keys.md).
+
+## Playbook configuration
+
+Here is the playbook configuration to set:
+
+![Playbook ExtractIP](docs/assets/playbooks/library/UseCases/ExtractIP.png)
```

Comment:
image is available [here](https://github.com/SEKOIA-IO/documentation/tree/main/docs/assets/playbooks/library/UseCases)
As this page is not yet linked in the documentation tree, it is not displayed

It will be displayed when this PR will be validated and that I link this page to the documentation